versions/draft-3/SPEC.md

```diff
@@ -1018,12 +1018,31 @@ task memory_test {
 
 ```
 $parameter_meta = 'parameter_meta' $ws* '{' ($ws* $parameter_meta_kv $ws*)* '}'
-$parameter_meta_kv = $identifier $ws* '=' $ws* $string
+$parameter_meta_kv = $identifier $ws* ':' $ws* $expression
 ```
 
-This purely optional section contains key/value pairs where the keys are names of parameters and the values are string descriptions for those parameters.
+This purely optional section contains key/value pairs where the keys are names of parameters and the values are literal expressions
+that describe those parameters. Note that the expressions *cannot* reference variables. The engine can ignore this section, with no loss of correctness. The extra information can be used, for example, to generate a user interface.
+
+> *Additional requirement*: Any key in this section MUST correspond to an input or output parameter.
 
-> *Additional requirement*: Any key in this section MUST correspond to a parameter in the command line
+For example:
+```wdl
+task wc {
+  File f
+  Boolean l = false
+  parameter_meta {
+    f : { "help": "Count the number of lines in this file" },
```

Comment:
Personal preference: I'd prefer something closer to the `object` literal or `runtime {}` block syntax.

Disclaimer: I probably won't do anything with these blocks so my aesthetic opinions on `parameter_meta` should be taken with a grain of salt

Reply:
Note that these kvs don’t match the grammar above: it wants ‘x: y’, this is ‘“x”: y’

Reply:
~~Note that these kvs don’t match the grammar above: it wants ‘x: y’, this is ‘“x”: y’~~

Edit: I got this wrong, but I think my mistake reinforces my preference above that whether we do or don't quote strings, we at least do it consistently.